pkg/lib/cockpit-components-logs-panel.jsx

```diff
@@ -130,7 +130,10 @@ export class LogsPanel extends React.Component {
     render() {
         return (
             <div className="panel panel-default cockpit-log-panel" role="table">
-                <div className="panel-heading">{this.props.title}</div>
+                <div className="panel-heading">
+                    <h2 className="panel-title">{this.props.title}</h2>
+                    <button className="link-button" role="link" onClick={e => cockpit.jump(this.props.goto_url)}>{_("All logs")}</button>
```

Comment:
If this property is not passed maybe don't show the button?
Else make sure that all places that use that Component pass that prop, and mark it as required in PropTypes.

Reply:
fixed